pkg/metal/core_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
```

Comment:
Wrong author

Reply:
VSCode carried this over from the `core.go` file. As we do not have an own header in this entire project, I just removed the header.